code/obj/item/clothing/uniforms.dm

```diff
@@ -897,18 +897,86 @@
 	icon_state = "referee"
 	item_state = "referee"
 
-/obj/item/clothing/under/shirt_pants
-	name = "shirt and pants"
-	desc = "A button-down shirt and some pants."
+//Black Pants
```

Comment:
//Brown Pants and //White Pants show up later down the line as well, they're there to make it easier to differentiate between all the variants. I think it might be worth keeping in for easier future reference though I'm clueless in code ettiquite so correct me if I'm wrong.

Reply:
"Section" comments are nice for keeping logical blocks of code together. This entire file (near enough) is basically just a big list of individual, technically unrelated items (with some groupings here and there). Basically, we don't have any official code style or anything that says one way or the other. I would at least make the comments the same way they're done elsewhere in the file (e.g. `// Brown Pants`, with a space after the `//`), so the file is consistent.